yt_dlp/extractor/youtube/_video.py

```diff
@@ -4029,6 +4029,11 @@ def process_language(container, base_url, lang_code, sub_name, client_name, quer
                     STREAMING_DATA_CLIENT_NAME: client_name,
                 })
 
+        def set_audio_lang_from_orig_subs_lang(lang_code):
```

Comment:
Wouldn't, say, `set_default_audio_lang()` have better described what it does? It's up to the caller where the `lang_code` comes froo.

Reply:
Sure